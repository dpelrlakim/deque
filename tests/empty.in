newDefault dq1
empty dq1
size dq1
newInit dq2
size dq2
empty dq2
= dq2 dq1
= dq1 dq2
empty dq1
empty dq2
