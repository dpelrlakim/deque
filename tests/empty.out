dq1:
1
dq1:
0
dq1:
dq1:
dq2: 2 3 5 7 11 13 17 19
8
dq1:
dq2: 2 3 5 7 11 13 17 19
0
dq1:
dq2: 2 3 5 7 11 13 17 19
dq1:
dq2:
dq1:
dq2:
1
dq1:
dq2:
1
dq1:
dq2:
